### ISSUE TITLE
video / mp4 and model preview 

### ISSUE THREAD

**Issue by biast12** [filegator/filegator#387]
So yeah it kinda says it, this is an enhancement request for video / mp4 preview

also one thing that would be really cool would be a model preview, have seen a few html libraries online that i think filegator could have good use for

**Comment by bmaximenko** [comment]
I migrated from h5ai, and found it disappointing that this feature is missing in filegator. In h5ai, visitors can watch/stream mp4 in a popup.

**Comment by biast12** [comment]
yeah i did the other way around, it's such a needed feature for filegator too, generally previewing makes such a tool so much more useful
